src/style.css

```
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: 50%;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url("./images/star-active.svg");
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url("./images/star-active.svg");
```
You can combine all the needed selectors as follow: